github.go

```diff
@@ -198,6 +199,62 @@ func (m *GithubClient) PostComment(prNumber, comment string) error {
 	return err
 }
 
+// GetChangedFiles ...
+func (m *GithubClient) GetChangedFiles(prNumber string, commitRef string) ([]ChangedFileObject, error) {
+	pr, err := strconv.Atoi(prNumber)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert pull request number to int: %s", err)
+	}
+
+	var cfo []ChangedFileObject
+
+	var filequery struct {
+		Repository struct {
+			PullRequest struct {
+				Files struct {
+					Edges []struct {
+						Node struct {
+							ChangedFileObject
+						}
+					} `graphql:"edges"`
+					PageInfo struct {
+						EndCursor   githubv4.String
+						HasNextPage bool
+					} `graphql:"pageInfo"`
+				} `graphql:"files(first:$changedFilesFirst, after: $changedFilesEndCursor)"`
+			} `graphql:"pullRequest(number:$prNumber)"`
+		} `graphql:"repository(owner:$repositoryOwner,name:$repositoryName)"`
+	}
+
+	offset := ""
+
+	for {
+		vars := map[string]interface{}{
+			"repositoryOwner":       githubv4.String(m.Owner),
+			"repositoryName":        githubv4.String(m.Repository),
+			"prNumber":              githubv4.Int(pr),
+			"changedFilesFirst":     githubv4.Int(100),
+			"changedFilesEndCursor": githubv4.String(offset),
+		}
+
+		if err := m.V4.Query(context.TODO(), &filequery, vars); err != nil {
+			return nil, err
+		}
+
+		for _, f := range filequery.Repository.PullRequest.Files.Edges {
+			cfo = append(cfo, ChangedFileObject{Path: f.Node.Path})
```

Comment:
I think you might be able to just append the `ChangedFileObject` directly here instead of redeclaring it? I.e.:

```go
cfo = append(cfo, f.Node.ChangedFileObject)
```

Or am I missing something?

Reply:
I think because append needs to resize the slice this is actually needed